packages/shared/src/components/MainLayout.tsx

```diff
@@ -231,36 +203,32 @@ function MainLayoutComponent({
         additionalButtons={additionalButtons}
         onLogoClick={onLogoClick}
       />
-      <Main
-        className={className}
-        isScreenCentered={isScreenCentered}
-        sidebarExpanded={sidebarExpanded}
-        isBannerAvailable={isBannerAvailable}
+      <main
+        className={classNames(
+          'flex flex-col tablet:pl-16 laptop:pl-11',
+          className,
+          !isScreenCentered && sidebarExpanded && 'laptop:!pl-60',
+          isBannerAvailable && 'laptop:pt-8',
+        )}
       >
-        {({ hydrated }) => {
```

Comment:
I think Ante added this for the reason of avoiding the SSR: false (which would be more ideal)
Maybe best to wait for his review :)

Reply:
Oh, cool. Yeah, I'd love to know more about why we'd choose one over the other so we can do consistent implementation.

Reply:
Looks like we have to, for this issue.

<img width="1562" alt="Screenshot 2024-10-23 at 4 34 27 PM" src="https://github.com/user-attachments/assets/106d11c0-fecd-48ee-84d7-d68820433bed">

Reply:
Ah right :), yeah think in general it's actually best if we could avoid SSR: false, but maybe you and Ante can sync about it tomorrow?

Reply:
Yeah, thank you for pointing out, I'll connect with him to know more about it 👀

Reply:
Yeah, I added the hack because of the above issue where sidebar does not know of the open state, its a rare case where we can't really do anything about it, except delay the render of the whole sidebar. `ssr: false` would just cause an overlap as shown in the picture above. 

@sshanzel let's sync tomorrow and see where we need the hack and where we can maybe fix it properly, I had a mind twister yesterday as well, two heads are smarter then one 💪